test/integration/unlogged_in_registered_user_must_login_to_checkout_test.rb

```diff
@@ -0,0 +1,35 @@
+require 'test_helper'
+
+class UnloggedInRegisteredUserMustLoginToCheckoutTest < ActionDispatch::IntegrationTest
+
+  test 'that an a registered user must log in to checkout' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.